src/Core/MailTemplates/Handlebars/FamiliesForEnterprise/FamiliesForEnterpriseSponsorshipReverting.html.hbs

```diff
@@ -0,0 +1,9 @@
+{{#>FullHtmlLayout}}
```

Comment:
What is the difference between this and `FamiliesForEnterpriseSponsorshipEnding`?

Reply:
Currently `Ending` doesn't get used at all. Since we are requiring payment information still for the creation of the families org I don't see a way that it would really get used. I will remove it for now.